assets/src/edit-story/elements/text/display.js

```diff
@@ -103,10 +131,37 @@ function TextDisplay({
       : '';
   });
 
+  if (backgroundType === 'highlight') {
```

Comment:
In the page preview, the text box doesn't render correctly (or at least not similarly to the on-canvas version:

Compare the initial text box in the page thumbnail in the carousel with the full background color to the highlight version after the switch (line-height seems to change completely breaking the layout?):

![preview](https://user-images.githubusercontent.com/637548/77368636-0998f400-6d33-11ea-84f1-0bc7b10436c9.gif)

Reply:
I did not work yet with that mini live preview. Thanks for point this out anyway 👍